src/platform/site-wide/sass/minimal.scss

```diff
@@ -37,25 +37,7 @@
 @import "~@department-of-veterans-affairs/css-library/dist/stylesheets/modules/m-button";
 
 // Formation utilities
-@import "~@department-of-veterans-affairs/formation/sass/utilities/background-color";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/border";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/color";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/display";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/flexbox";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/font-family";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/font-size";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/font-style";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/font-weight";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/height-width";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/line-height";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/margins";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/measure";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/padding";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/position";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/text-align";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/text-decoration";
-@import "~@department-of-veterans-affairs/formation/sass/utilities/visibility";
-
+@import "~@department-of-veterans-affairs/css-library/dist/stylesheets/utilities";
```

Comment:
I did a quick search for this because I thought pre-check-in was using minimal.scss. It was, once upon a time, but no longer is. I wonder if we actually even need this anymore :thinking:

Reply:
Doing a quick compare with removing the import it doesn't look like anything changes on the page, we will need to verify with vaos. For now we are reverting this change.